source/extensions/filters/common/rbac/engine_impl.cc

```diff
@@ -23,11 +23,11 @@ bool RoleBasedAccessControlEngineImpl::allowed(const Network::Connection& connec
                                                std::string* effective_policy_id) const {
   bool matched = false;
 
-  for (auto it = policies_.begin(); it != policies_.end(); it++) {
-    if (it->second.matches(connection, headers, metadata)) {
+  for (const auto& policie : policies_) {
```

Comment:
a/policie/policy

Reply:
Hmm, wonder why spelling CI didn't catch this when it caught another one that was very similar: https://github.com/envoyproxy/envoy/pull/7790/commits/5203b0ec15a332ec658b88461d6598eb572a78d5